server/public/model/post.go

```diff
@@ -207,6 +207,21 @@ type SearchParameter struct {
 	IncludeDeletedChannels *bool   `json:"include_deleted_channels"`
 }
 
+func (sp SearchParameter) Auditable() map[string]interface{} {
+	return map[string]interface{}{
+		"terms":                    sp.Terms,
```

Comment:
Do we need to resolve the pointer or will mlog take care of that?

Reply:
Logr handles that.